examples/kafka-hub/helm/README.md

```diff
@@ -0,0 +1,150 @@
+# [Kafka Websubhub] Helm charts
```

Comment:
Add the link?

Reply:
In the overview section we have already added the link. Hence will remove the link notation from here:
- [ccf6866aeee9e6c5977d521dea7fd5e456196fa9](https://github.com/ballerina-platform/module-ballerina-websubhub/pull/1076/commits/ccf6866aeee9e6c5977d521dea7fd5e456196fa9)